app/cdap/components/ConnectionList/Components/SubHeader/__test__/Breadcrumb.test.tsx

```diff
@@ -17,9 +17,19 @@
 import React from 'react';
 import { render, screen } from '@testing-library/react';
 import BreadCumb from '../index';
+import history from 'services/history';
+import { Router, Route } from 'react-router';
 
-test('renders BreadCumb Component', () => {
-  render(<BreadCumb />);
-  const ele = screen.getByTestId(/bread-comb-container-parent/i);
-  expect(ele).toBeInTheDocument();
+describe('renders BreadCumb Component', () => {
+  render(
+    <Router history={history}>
+      <Route>
+        <BreadCumb />
+      </Route>
+    </Router>
+  );
+  it('should render the Breadcrumb component', () => {
+    const ele = screen.getByTestId(/bread-comb-container-parent/i);
```

Comment:
bread crumb is misspelled

Reply:
is this a regex? Can yo use a string here to do the matching instead

Reply:
updated